app/models/shopify_import/base.rb

```diff
@@ -1,40 +1,34 @@
 module ShopifyImport
   class Base
-    def initialize(credentials: {}, client: nil)
```

Comment:
@vfonic I am not sure of moving it out of here. As any object here should be able to connect to Shopify as AR model can connect to DB 
https://github.com/rails/activeresource/blob/master/lib/active_resource/base.rb .
Moving it out of here could cause a lot of problems with future jobs and re-imports. If connection will be done in Invoker what how long it can be valid? What if import will go much longer? If we really would like to move it from here we also have to have some connection here.